src/main/java/com/ibm/disni/rdma/verbs/impl/NatRegMrCall.java

```diff
@@ -63,10 +63,10 @@ public void set(IbvPd pd, long address, int length, int access) {
 		this.access = access;
 		
 		if (cmd != null){
-			cmd.free();
-			cmd = null;
+			cmd.getBuffer().clear();
```

Comment:
This seems to make sense, no need to free and null the cmd objects, but just clear the buffer. But it seems if we decide to do this, then we should consistently make that change in this PR to all other SVC objects. Currently other SVC objects (e.g., NatPostSendCall) still implement the free and null. Any reason you didn't change those?

Reply:
But this buffer is always of the same size. In other objects buffers can be of different sizes, based on the list length of wcList

Reply:
How? the size of the WR array is fixed within one SVC object and is not going to change. If an application wants to post a different set of WRs or a longer list of WRs it will need to create a SVC object (e.g., NatPostSendCall).